debian/control

```diff
@@ -0,0 +1,43 @@
+Source: machine-emulator
+Section: libs
+Priority: optional
+Maintainer: root <root@unknown>
+Rules-Requires-Root: no
+Build-Depends:
+ debhelper-compat (= 13),
+ libboost1.81-dev,
+ liblua5.4-dev,
+ libslirp-dev,
+ lua-dkjson,
+ lua-lpeg,
+ lua-posix-dev,
+ lua-sec-dev,
+ lua-socket-dev,
```

Comment:
`lua-posix`, `lua-sec`, `lua-socket` are only needed for testing could remove if we are not performing tests.
`lua-lpeg` can be removed
`lua-dkjson` we are not using anymore for now

Reply:
I took them from the Dockerfile, I'll add an issue to clean it up as well.